aten/src/ATen/native/native_functions.yaml

```diff
@@ -3101,6 +3101,10 @@
   dispatch:
     CompositeExplicitAutograd: mkldnn_convolution
 
+- func: mkldnn_convolution_transpose(Tensor self, Tensor weight, Tensor? bias, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups) -> Tensor
```

Comment:
This entry is not needed. The others functions are still here for BC reasons and will be removed as well soon.
We expect to only have `convolution` and `convolution_backward` here in the end state.

Reply:
both the foward and backward of mkldnn_convolution_transpose entries have been removed from `native_functions.yaml`, use stub instead.